src/sentry/demo/demo_start.py

```diff
@@ -172,21 +174,37 @@ def get_one_release(org: Organization, project_slug: Optional[str]):
     return f"/organizations/{org.slug}/releases/{version}/?project={project.id}"
 
 
-def get_one_issue(org: Organization, project_slug: Optional[str]):
+def get_one_issue(org: Organization, project_slug: Optional[str], error_type: Optional[str]):
     group_query = Group.objects.filter(project__organization=org)
-    if project_slug:
+
+    if project_slug and error_type:
```

Comment:
@roggenkemper I think we could simplify this by having a `if project_slug` block then add an `if error_type` inside